### PR TITLE
OSDOCS-7749: [MOBB F2F] Migrate MOBB Prerequisites Checklist to Deploy ROSA Cluster with STS to ROSA product documentation

### PR DESCRIPTION
**Pull request by bmcelvee**
Version(s):
4.13+ 

Issue:
https://issues.redhat.com/browse/OSDOCS-7749

Link to docs preview:
Perquisites checklist: https://64789--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs#rosa-mobb-prereq-checklist_rosa-sts-aws-prereqs
Tutorial: https://64789--docspreview.netlify.app/openshift-rosa/latest/rosa_tutorials/rosa-mobb-prerequisites-tutorial

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/rosa-mobb-prereq-checklist.adoc [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+//date: '2023-07-27'
+//title: Prerequisites Checklist to Deploy ROSA Cluster with STS 
+//tags: ["ROSA", "STS"]
+//authors:
+//  - Byron Miller
+//  - Connor Wooley
+//  - Diana Sari
+//---
+
+This is a checklist of prerequisites needed to spin up a classic [product-title] cluster with links:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[STS].
```
Comment (paulczar) [inline modules/rosa-mobb-prereq-checklist.adoc:23]: this didn't render correctly on the preview url

image

File: modules/rosa-mobb-prereq-checklist.adoc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+- The hosts on the network that you provision.
+- The internet to obtain installation media.
+
+== Accounts and CLIs Prerequisites
+
+Accounts and CLIs you must install to deploy the cluster.
+
+=== AWS account:
+  - Gather the following details:
+      - AWS IAM User
```
Comment (paulczar) [inline modules/rosa-mobb-prereq-checklist.adoc:42]: indents not working right in preview

image

File: modules/rosa-mobb-prereq-checklist.adoc [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+[source,terminal]
+----
+$ rosa whoami
+----
++
+. Ensure you have sufficient quota by running: 
++
+[source,terminal]
+----
+rosa verify quota
```
Comment (EricPonvelle) [inline modules/rosa-mobb-prereq-checklist.adoc:124]: Need a $ here

File: modules/rosa-mobb-prereq-checklist.adoc [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+----
++
+* See link:https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html#rosa-aws-policy-provisioned_rosa-sts-aws-prereqs[Provisioned AWS Infrastructure] for more details on AWS services provisioned for ROSA cluster. 
+* See link:https://docs.openshift.com/rosa/rosa_planning/rosa-sts-required-aws-service-quotas.html[Required AWS service quotas] for more details on AWS services quota. 
+
+=== OpenShift CLI (`oc`)
+
+. Install from [here](https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html) or from the OpenShift console [here](https://console.redhat.com/openshift/downloads#tool-oc).
```
Comment (EricPonvelle) [inline modules/rosa-mobb-prereq-checklist.adoc:132]: These links are still in Markdown. Should be:

* link:https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html[here]
* link:https://console.redhat.com/openshift/downloads#tool-oc[here]

File: modules/rosa-mobb-prereq-checklist.adoc [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+** The subnet CIDR blocks must belong to the machine CIDR that you specify.
+* Set both `enableDnsHostnames` and `enableDnsSupport` to `true`.
+** That way, the cluster can use the Route 53 zones that are attached to the VPC to resolve cluster internal DNS records.
+* Verify route tables by running: 
++
+[source,terminal]
+ ----
+ $ aws ec2 describe-route-tables --filters "Name=vpc-id,Values=<vpc-id>"
+ ----
+ +
```
Comment (EricPonvelle) [inline modules/rosa-mobb-prereq-checklist.adoc:189]: This doesn't render correctly.
image

I think you can delete the `+` at 189 and it should work.

File: rosa_tutorials/rosa-mobb-prerequisites-tutorial.adoc [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+----
+$ aws configure
+----
++
+. You will be required to enter an `AWS Access Key ID` and an `AWS Secret Access Key` along with a default region name and output format:
++
+[source,terminal]
+----
+$ aws configure
+AWS Access Key ID []:
```
Comment (EricPonvelle) [inline rosa_tutorials/rosa-mobb-prerequisites-tutorial.adoc:111]: Lines 111 - 114 should be that `.Sample output` style.

**Review by EricPonvelle — COMMENTED** [review]
Minor things

**Review by paulczar — APPROVED** [review]

**Pull request merged**